# These values will range from -1 to 1 to represent the thruster values.
# -1 represents full speed reverse, 1 represents full speed ahead,
# 0 represents dead stop, floats represent intermediary values.
float32 thruster_front
float32 thruster_back
float32 thruster_topleft
float32 thruster_topright
float32 thruster_backleft
float32 thruster_backright
